server/models/menu.js

```diff
@@ -0,0 +1,63 @@
+let knex = require('../database/knex.js');
+
+
+
+
+exports.getMenu = async (req, res) => {
+    let menus = await knex.select('image_url').from('foods');
+    return JSON.parse(JSON.stringify(menus));
+};
+
+
+
+//เดี๋ยวเปลี่ยนเลข39เป็นตัวแปร
+//39 คือ จำนวน id ของอาหารทั้งหมด
+exports.getRandomMenu = async (req, res) => {
+    let arrayUrl = [];
+    
+    for(i = 0; i < 6; i++)
+        do{
+           arrayUrl[i] = Math.floor(Math.random() * 39)+1;
+           console.log(arrayUrl[i]);
+        } while(new Set(arrayUrl).size !== arrayUrl.length);
+
+    let menus = await knex.select('image_url').whereIn('id', arrayUrl).from('foods');
+    return JSON.parse(JSON.stringify(menus));
```

Comment:
Why you need to use **do-while**? and if random duplication occurs, how do you prevent it?

Reply:
The code ```new Set(arrayUrl).size !== arrayUrl.length``` will return 1 if there is duplication